ppinot4py/computers/conditions_computer.py

```diff
@@ -21,15 +22,20 @@ def condition_computer(dataframe, id_case, condition):
     return filteredSeries
 
 
-
-def _time_instant_condition_resolve(dataframe, id_case, condition: TimeInstantCondition):
+def _time_instant_condition_resolve(dataframe, id_case, condition: TimeInstantCondition, activity_column, transition_column):
     TimeInstantCondition(RuntimeState.START, AppliesTo.PROCESS)
     if condition.applies_to == AppliesTo.DATA:
         return _time_instant_condition_data_resolve(dataframe, id_case, str(condition.changes_to_state))
     elif condition.applies_to == AppliesTo.PROCESS:
         return _time_instant_condition_process_resolve(dataframe, id_case, condition.changes_to_state)
+    elif condition.applies_to == AppliesTo.ACTIVITY:
+        if(check_column_existence(dataframe, activity_column, transition_column)):
+            data_condition = f'`{activity_column}` == {condition.activity_name} and `{transition_column}` == {condition.changes_to_state}'
```

Comment:
Does this work if the dataset do not have a transition_column? In that case, it should be possible to do it if condition.changes_to_state == RuntimeState.END because it is the default behaviour